### DISCUSSION TITLE
[1817800]火花奇遇记：自动化冒险 {20240429更新}

### DISCUSSION THREAD

**Discussion by zhuifengzhe1** [pjy612/SteamManifestCache#630] [Share]
### Steam 游戏ID

`1817800`

### 密钥列表

```hash
1817801;b59ebe145d953e10adb462fd930cbf0da37483e64c0896babf1861638f87b6a5
229007;eba9fa9ff4f811929c119074450f49f739113bcae9317ded9bea262aa21ec58d
```

### 清单附件

[depotcache.zip](https://github.com/pjy612/SteamManifestCache/files/15148750/depotcache.zip)

### 备注
Code:39FD614C3C9F9E5A
Email:[asdadad331@outlook.com](mailto:asdadad331@outlook.com)

**Comment by zhuifengzhe1** [comment]
229007是这个：.NET 4.8 Redist

**Reply by pjy612** [reply]
https://steamdb.info/app/1817800/depots/
任何号 Shared Install 都是能自动获取的，所以不用传
